commit 42804ac491a4deb2a7460558a0fe6921f9c27bfc
Author: Hannes Frederic Sowa <hannes@stressinduktion.org>
Date:   Mon Mar 13 00:01:30 2017 +0100

    dccp: fix memory leak during tear-down of unsuccessful connection request
    
    [ Upstream commit 72ef9c4125c7b257e3a714d62d778ab46583d6a3 ]
    
    This patch fixes a memory leak, which happens if the connection request
    is not fulfilled between parsing the DCCP options and handling the SYN
    (because e.g. the backlog is full), because we forgot to free the
    list of ack vectors.
    
    Reported-by: Jianwen Ji <jiji@redhat.com>
    Signed-off-by: Hannes Frederic Sowa <hannes@stressinduktion.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1407_42804ac491a4deb2a7460558a0fe6921f9c27bfc.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1407_42804ac491a4deb2a7460558a0fe6921f9c27bfc.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
